apps/prairielearn/src/cron/calculateAssessmentQuestionStats.js

```diff
@@ -1,21 +1,19 @@
-const util = require('util');
+// @ts-check
+import { callbackify } from 'util';
 
-const { logger } = require('@prairielearn/logger');
-const sqldb = require('@prairielearn/postgres');
+import { logger } from '@prairielearn/logger';
+import { loadSqlEquiv, queryRows } from '@prairielearn/postgres';
 
-const sql = sqldb.loadSqlEquiv(__filename);
+import { updateAssessmentQuestionStatsForAssessment } from '../lib/assessment';
+import { IdSchema } from '../lib/db-types';
 
-module.exports = {};
+const sql = loadSqlEquiv(__filename);
 
-module.exports.run = function (callback) {
-  util.callbackify(async () => {
-    const result = await sqldb.queryAsync(sql.select_assessments, {});
-    const assessments = result.rows;
-    for (const assessment of assessments) {
-      logger.verbose(
-        `calculateAssessmentQuestionStats: processing assessment_id = ${assessment.id}`,
-      );
-      await sqldb.callAsync('assessment_questions_calculate_stats_for_assessment', [assessment.id]);
-    }
-  })(callback);
-};
+export async function runAsync() {
+  const assessment_ids = await queryRows(sql.select_assessments, {}, IdSchema);
```

Comment:
Nit: you can actually omit the params argument if it's not needed.

```suggestion
  const assessment_ids = await queryRows(sql.select_assessments, IdSchema);
```